### PR TITLE
[@types/mimetext] Add missing MIME types to setAttachment

### PR DESCRIPTION
**Pull request by hokyunrhee**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/mimetext/index.d.ts [hunk 1]
```diff
@@ -31,6 +31,82 @@ export type MailLocation = string | MailLocationData;
...
+    | 'video/3gpp'
+    | 'video/3gpp2'
+    | 'video/mp2t'
+    | 'video/mp4'
+    | 'video/mpeg'
+    | 'video/ogg'
+    | 'video/webm'
+    | 'video/x-msvideo';
+
+export type MIMEtype = ApplicationType | AudioType | FontType | ImageType | TextType | VideoType;
```
Comment (JoshuaKGoldberg) [inline types/mimetext/index.d.ts:108]: Nit: this should probably keep the acronym + title casing the rest of the type use.

```suggestion
export type MIMEType = ApplicationType | AudioType | FontType | ImageType | TextType | VideoType;
```

**Review by JoshuaKGoldberg — APPROVED** [review]
LGTM!

File: types/mimetext/index.d.ts (hunk 1 above)
Comment (hokyunrhee) [reply types/mimetext/index.d.ts]: Tanks for your feedback.

**Review by JoshuaKGoldberg — APPROVED** [review]
💯

**Comment by hokyunrhee** [comment]
Ready to merge

**Pull request merged**